src/AzureClient/AzureClient.cs

```diff
@@ -502,6 +502,30 @@ public async Task<ExecutionResult> GetJobListAsync(IChannel channel, string filt
             return jobs.ToExecutionResult();
         }
 
+        public async Task<ExecutionResult> GetQuotaListAsync(IChannel channel)
+        {
+            if (ActiveWorkspace == null)
+            {
+                channel.Stderr($"Please call {GetCommandDisplayName("connect")} before reading quota information.");
+                return AzureClientError.NotConnected.ToExecutionResult();
```

Comment:
IIRC, you'll want to pass `ExecuteStatus.Error` as an optional parameter to `ToExecutionResult` to mark this as an error.

Reply:
For this one, I'm following the pattern of the other methods in the client, and they appear to be leaving the default when converting errors to execution results. Keeping as is for consistency for now, but we can chat offline about this.